docs/plugins/getting_started/finding_and_installing_plugins.md

```diff
@@ -1,5 +1,5 @@
 (find-and-install-plugins)=
-# Finding and installing a napari plugin
+# Finding and installing plugins
```

Comment:
I can't comment on lines below, but because you have already disambiguated unpublished plugins vs. from the hub, I'd remove the section below on "Finding plugins on the napari hub".

Reply:
I've removed that section ("Finding plugins on the napari hub")